baekjoon_7/5622.py

```diff
@@ -0,0 +1,14 @@
+# 5622 - 다이얼
+# 딕셔너리를 쓰면 편하게 풀리지 않을까?
+dial = {'ABC': 3, 'DEF': 4, 'GHI': 5, 'JKL': 6, 'MNO': 7, 'PQRS': 8, 'TUV': 9, 'WXYZ': 10}  # 딕셔너리 생성
+
+wrd = input()
+
+time = 0
+# 여기선 구글링의 도움을 받았습니다 :( for 문 안에 for 문을 넣어 돌린다는게 약간 생각하기 어렵더라구요.
+for i in wrd:  # 입력된 단어를 분해
+    for j in dial.keys():  # j 로 딕셔너리의 키값을 하나씩 빼 와서
```

Comment:
for문을 통해 각 key값만 얻어오는게 목적이라면 굳이 `.keys()`함수를 호출할 필요가 없어요
```suggestion
    for j in dial:  # j 로 딕셔너리의 키값을 하나씩 빼 와서
```
도 가능해용.